packages/cdk8s-cli/BUILD.bazel

```diff
@@ -0,0 +1,116 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@build_bazel_rules_nodejs//:index.bzl", "copy_to_bin")
+load("//tools:defaults.bzl", "jest_test", "pkg_npm", "ts_library")
+load("//tools:package-json/package_json.bzl", "package_json")
+
+package_json(
+    name = "package_json",
+    package_name = "cdk8s-cli",
+    bin = {
+        "cdk8s": "bin/cdk8s.js",
+    },
+    desc = "CDK for Kubernetes CLI",
+    experimental = True,
+    peer_deps = {
+        "codemaker": "^0.22.0",
+        "yargs": "^15.1.0",
```

Comment:
I don't think these should be peer deps. I think it was a mistake in our original package.json

Reply:
Sounds like I missed that in one of the rebases. I'll remove based on the current spec.